pkg/flowaggregator/flowaggregator_test.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package flowaggregator
+
+import (
+	"testing"
+	"time"
+
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/assert"
+
+	ipfixentities "github.com/vmware/go-ipfix/pkg/entities"
+	ipfixintermediate "github.com/vmware/go-ipfix/pkg/intermediate"
+	ipfixregistry "github.com/vmware/go-ipfix/pkg/registry"
+
+	ipfixtest "github.com/vmware-tanzu/antrea/pkg/ipfix/testing"
+)
+
+const (
+	testTemplateID      = uint16(256)
+	testExportFrequency = 60 * time.Second
+)
+
+func TestFlowAggregator_sendTemplateRecord(t *testing.T) {
```

Comment:
Please add a TODO that we will add test for sendDataRecord when we support adding multiple records to single set. Currently, we are supporting adding only one record from one flow key to the set.